clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -565,6 +565,14 @@ class CIRGenBuilderTy : public mlir::OpBuilder {
                                      mlir::cir::CastKind::floating, v);
   }
 
+  mlir::Value createFPTrunc(mlir::Value v, mlir::Type destType) {
```

Comment:
Code for this function is currently identical to `createFPExt`. But I think it makes sense to keep this functions separate.